port/aix/omrosdump_helpers.c

```diff
@@ -173,9 +176,10 @@ genSystemCoreUsingGencore(struct OMRPortLibrary *portLibrary, char *filename)
 	rc = gencore(&coreDumpInfo);
 
 #if defined(DUMP_DBG)
+	savedErrno = errno;
 	portLibrary->tty_printf(portLibrary, "\tomrdump_create: gencore/coredump returned: %i\n", rc);
 	if (rc == -1) {
-		portLibrary->tty_printf(portLibrary, "\tomrdump_create: errno: %u %s\n", errno, portLibrary->error_last_error_message(portLibrary));
+		portLibrary->tty_printf(portLibrary, "\tomrdump_create: errno: %i %s\n", savedErrno, portLibrary->error_last_error_message(portLibrary));
 	}
 	fflush(stdout);
 #endif
```

Comment:
Some of this code is old, and predates the coding standards. Can I also request you to update the `#endif` comments for all `defined(DUMP_DBG)`? There are only 5 locations in this file. As we modify old code, we want to enforce as much of the coding standards as possible. Also, it will be more consistent.

Reply:
Sure. Done.